graphblas/matrix.py

```diff
@@ -136,16 +141,39 @@ def _deserialize(pieces, name):
 
     @property
     def S(self):
+        """
+        Creates a Mask based on the structure of the Matrix.
+        """
         return StructuralMask(self)
 
     @property
     def V(self):
+        """
+        Creates a Mask based on the values in the Matrix (treating each value as truthy).
+        """
         return ValueMask(self)
 
     def __delitem__(self, keys):
+        """
+        Delete a single element, row/column, or submatrix.
+
+        .. code-block:: python
+
+            del M[1, 5]
```

Comment:
Can we instead use `>>>` for code blocks?

Reply:
Most of what I'm showing are complementary one-line statements (method vs functional) with no output.

I personally like this look better.
![CurrentNotation](https://user-images.githubusercontent.com/2807270/183739062-45348aa0-108d-4f6a-b560-bdca2e00eb1f.png)

This just feels awkward and must be split into two separate code blocks because comments don't make sense when using `>>>` command-prompt style.
![UseTripleLT](https://user-images.githubusercontent.com/2807270/183739069-8623da02-cbbb-45da-8b37-dde083503e84.png)

Reply:
Docstrings aren't just read from the website though.  Reading via a REPL or the file is also very common.

An argument for the RST version is that it is friendlier to copy/paste wherever it is.

Reply:
So we can use `.. code-block:: python` in docstrings if you'd like.  It does look better when rendered.